tests/integration/test_multiple_disks/test.py

```diff
@@ -1135,6 +1135,70 @@ def optimize(num):
         for node in [node1, node2]:
             node.query("DROP TABLE IF EXISTS replicated_table_for_moves SYNC")
 
+def test_simple_replication_and_moves_on_cluster(start_cluster):
```

Comment:
I think it's better to edit the original test than to copy-paste.

Reply:
OK.